iOS/JSQMessagesViewController/externals/Makefile

```diff
@@ -8,20 +8,10 @@ JSQ_TARGET=JSQMessagesViewControllerLib
 
 all :  $(JSQ_TARGET)/lib$(JSQ_TARGET).a ios/Resources/JSQMessagesViewController.nib
 
-
-ios/JSQSystemSoundPlayer/JSQSystemSoundPlayer/Classes/JSQSystemSoundPlayer.h :
-	mkdir -p ios
-	cd ios && git clone git@github.com:jessesquires/JSQSystemSoundPlayer.git
-	cd ios/JSQSystemSoundPlayer && git checkout 2.0.1
-
-ios/JSQMessagesViewController/JSQMessagesViewController/JSQMessages.h : ios/JSQSystemSoundPlayer/JSQSystemSoundPlayer/Classes/JSQSystemSoundPlayer.h 
+ios/JSQMessagesViewController/JSQMessagesViewController/JSQMessages.h :
```

Comment:
@bholmes Ok, I found a reproducible scenario and reverted the change. It should be all good now.